src/main/webapp/WEB-INF/classes/.gitignore

```diff
@@ -0,0 +1 @@
+/META-INF/
```

Comment:
Hi Jin ,

There is this file src/main/webapp/dev/WEB-INF/classes/.gitignore that gets
deleted automatically when ever i edit the file ./mashup.jsp.
This is the same problem i had the last time. I have not deleted this file.
I dont know why this file gets deleted automatically like this .
Just hope you will answer me very soon and let me know what to do. I have
over stayed on this issue .
Thanks

On Wed, Apr 8, 2015 at 12:03 PM, tchuenkam ulrich <ulrichmobi2@gmail.com>
wrote:

> Ok Jin ,
> Will just do as you said .
>  thanks.
>
> On Wed, Apr 8, 2015 at 3:04 AM, Kang Hong Jin <notifications@github.com>
> wrote:
>
>> In src/main/webapp/WEB-INF/classes/.gitignore
>> <https://github.com/TEAMMATES/repo/pull/3157#discussion_r27937773>:
>>
>> > @@ -0,0 +1 @@
>> > +/META-INF/
>>
>> Since there are so many irrelevant commits in this pull request, perhaps
>> make a new pull request? This time, be sure to not include the unrelated
>> commits and files.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/TEAMMATES/repo/pull/3157/files#r27937773>.
>>
>
>
>
> --
> ulrich tchuenkam
>



-- 
ulrich tchuenkam